src/utils/utils.ts

```diff
@@ -64,3 +64,182 @@ export function settingsListToTree(list: Setting[]): Tree {
   });
   return root;
 }
+
```

Comment:
remove they are already in the metrics.ts file

Reply:
and/or move your new functions to the other file. 
localstorage files are in localStorage.ts

Reply:
done

Reply:
No local storage was moved aswell, it shouldnt be added here.

Reply:
not sure what to remove here as these aren't my functions. could someone have a look as not remove something needed

Reply:
All of the local storage functions seen here in green